external/llvm-project/mlir/lib/Conversion/GPUCommon/GPUToLLVMConversion.cpp

```diff
@@ -1333,8 +1333,9 @@ LogicalResult ConvertSetDefaultDeviceOpToGpuRuntimeCallPattern::matchAndRewrite(
     gpu::SetDefaultDeviceOp op, OpAdaptor adaptor,
     ConversionPatternRewriter &rewriter) const {
   Location loc = op.getLoc();
-  setDefaultDeviceCallBuilder.create(loc, rewriter, {adaptor.getDevIndex()});
-  rewriter.replaceOp(op, {});
```

Comment:
Nobody else in MLIR passes {} to replaceOp().

Reply:
I think the original intention was to delete `op`, but I guess that since the `setDefaultDeviceCall` is returning nothing, this should be equivalent.